vignettes/how-to/use-bslib.Rmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "How-to: Use bslib with Rhino"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How-to: Use bslib with Rhino}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Introduction
+
+The `bslib` R package is a UI toolkit based on Bootstrap.
+Shiny uses it under the hood for its default appearance,
+but it's also possible to use `bslib` directly to gain some benefits:
+1. Additional components (e.g. value boxes, accordions, tooltips).
+2. Customization and theming.
+3. A refined interface (e.g. `bslib::page_sidebar` instead of `shiny::sidebarLayout`).
+4. Access to the latest Bootstrap (v5).
+
+Rhino comes with built-in support for writing custom Sass code in `app/styles/main.scss`.
+This guide explains how to combine `bslib` and custom Sass code.
+
+If you don't want to write any custom Sass,
```

Comment:
Please make this visually emphasized (maybe a box around the text or  a background?).